browser/ui/browser_commands.cc

```diff
@@ -237,7 +237,9 @@ void NewOffTheRecordWindowTor(Profile* profile) {
     return;
   }
 
+#if BUILDFLAG(ENABLE_TOR)
```

Comment:
Should this guard be around both definitions for `NewOffTheRecordWindowTor`? I think the places calling into here already have `ENABLE_TOR` guards. Or if the callers are missing, we should add the guards there too
- https://github.com/brave/brave-core/blob/1808746eec7a133ddf6934dae05ae8a1b0b2ecd9/browser/ui/brave_browser_command_controller.cc#L562-L564
- https://github.com/brave/brave-core/blob/1808746eec7a133ddf6934dae05ae8a1b0b2ecd9/browser/brave_app_controller_mac.mm#L249-L254
- etc

This might help ID other places we missed

Reply:
Updated in the latest